src/modules/Channel/components/SuggestedMentionList/index.tsx

```diff
@@ -56,7 +56,17 @@ function SuggestedMentionList(props: SuggestedMentionListProps): JSX.Element {
   const [searchString, setSearchString] = useState('');
   const [lastSearchString, setLastSearchString] = useState('');
   const [currentUser, setCurrentUser] = useState<User>(null);
-  const [currentMemberList, setCurrentMemberList] = useState<Array<Member>>([]);
+  const [currentMemberList, setCurrentMemberList] = useState<Member[]>([]);
+
+  function getCurrentSearchedMemberList(): Member[] {
+    return currentChannel.members
+      .sort((a, b) => a.nickname?.localeCompare(b.nickname))
+      .filter(
+        (member) => member.nickname?.toLowerCase().startsWith(searchString.toLowerCase())
+        && member.userId !== currentUser?.userId
+        && member.isActive,
+      ).slice(0, maxSuggestionCount);
+  }
```

Comment:
Since it doesn't get a parameter and manipulate the param data, probably it's better to define as const \w useMemo(). 
```suggestion
  const currentSearchedMemberList(): Member[] => useMemo(() => {
    return currentChannel.members
    .sort((a, b) => a.nickname?.localeCompare(b.nickname))
    .filter(
      (member) =>
        member.nickname?.toLowerCase().startsWith(searchString.toLowerCase()) &&
        member.userId !== currentUser?.userId &&
        member.isActive,
    ).slice(0, maxSuggestionCount);
  }, [currentChannel.members.length])
```